src/game.zig

```diff
@@ -84,10 +84,25 @@ pub const Player = struct {
 		main.renderer.MeshSelection.placeBlock(&inventory__SEND_CHANGES_TO_SERVER.items[selectedSlot]);
 	}
 
-	pub fn breakBlock() void { // TODO: Breaking animation and tools
+	pub fn breakBlock() void { // TODO: Breaking animation and tools 
 		if(!main.Window.grabbed) return;
 		main.renderer.MeshSelection.breakBlock(&inventory__SEND_CHANGES_TO_SERVER.items[selectedSlot]);
 	}
+
+	pub fn acquireSelectedBlock() void { //acquireSelectedBlock
+		if (main.renderer.MeshSelection.selectedBlockPos) |selectedPos| {
+			const block = main.renderer.mesh_storage.getBlock(selectedPos[0], selectedPos[1], selectedPos[2]) orelse return;
+			for (0..items.itemListSize) |idx|{
+				if (items.itemList[idx].block) |_block|{
+					if (_block == block.typ){
```

Comment:
I'm pretty sure you could also do the comparison directly `if(items.itemList[idx].block == block.typ)` instead of making two ifs.